pages/add-post.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <v-row class="my-2" align="center" justify="center" style="height: 100%">
+    <v-col sm="8" md="8" lg="8" xl="6">
+      <v-card class="pa-10" elevation="2" outlined style="border-radius: 20px">
+        <v-card-title class="justify-center">Formularz ogłoszenia</v-card-title>
+        <v-radio-group v-model="type">
+          <template #label>
+            <div>Wybierz <strong>typ ogłoszenia</strong></div>
+          </template>
+          <v-divider class="mb-2"></v-divider>
+          <v-radio label="przedmiot" value="gift"></v-radio>
+          <v-radio label="zbiórka" value="fundraising"></v-radio>
+        </v-radio-group>
+        <v-expand-transition>
+          <v-slider
+            v-if="type == 'fundraising'"
+            v-model="price"
+            :min="minPrice"
+            :max="maxPrice"
+          ></v-slider>
+        </v-expand-transition>
+        <v-expand-transition>
+          <v-text-field
+            v-if="type == 'fundraising'"
+            v-model="price"
+            label="cena"
+            :rules="priceRules"
+            suffix="zł"
+            outlined
+            dense
+            required
+          ></v-text-field>
+        </v-expand-transition>
+        <v-form ref="form" v-model="valid" lazy-validation>
+          <v-text-field
+            v-model="url"
+            label="link do zbiórki"
+            outlined
+            dense
+          ></v-text-field>
+          <v-text-field
+            v-model="title"
+            :counter="100"
+            :rules="titleRules"
+            label="tytuł"
+            outlined
+            dense
+            required
+          ></v-text-field>
+          <v-textarea
+            v-model="desc"
+            label="opis"
+            height="300"
+            :rules="descRules"
+            no-resize
+            outlined
+            dense
+            required
+          ></v-textarea>
+          <div class="d-sm-flex">
+            <v-file-input
+              v-model="files"
+              small-chips
+              multiple
+              label="zdjęcia"
+              max="2"
```

Comment:
If that's a limit on the amount of files sent, it seems to not work. Also, 2 images seem like a pretty low number. That's something to discuss, but I guess 5 (or even 10?) would be better?